### DISCUSSION TITLE
Noob question regarding non-live cams

### DISCUSSION THREAD

**Discussion by stain3565** [blakeblackshear/frigate#4439] [Ask A Question]
As a noob, I have several issues currently but will ask questions separately.

This one is a fairly simple question.

As well as cams that should integrate with live streams (not all working but that is for another discussion), I have one that is wireless battery powered, that sends Motion detected video to an email address and I have processes that extract these to a NAS. Although I realise that this cannot be a "live" cam in frigate,it would be good to at least have the captured video represented in the interface. Is this possible?

**Comment by NickM-27** [comment]
frigate is designed to work with cameras that have a 24/7 RTSP stream. It may be technically possible to get pre-recorded video into the UI (ffmpeg can read from mp4 files), it would be quite difficult and I don't think it would make much sense since that's not really the purpose of frigate.
